ApolloSQLite/SQLiteNormalizedCache.swift

```diff
@@ -88,7 +88,7 @@ public final class SQLiteNormalizedCache: NormalizedCache {
   }
 }
 
-private let serializedReferenceKey = "reference"
+private let serializedReferenceKey = "$reference"
```

Comment:
Yeah, you're right, I wasn't thinking clearly, there are no nested objects in records of course. Although we may want to leave open the possibility of storing JSON in a field if something like that becomes part of the GraphQL spec? May be too forward looking.